data/help/texts.json

```diff
@@ -100,7 +100,7 @@
     "order": 8,
     "name": "<i|I>: Radioactivity and mutation",
     "messages": [
-      "Though it is relatively rare, certain areas of the world may be contaminated with radiation.  It will gradually accumulate in your body, weakening you more and more.  While in radiation-free areas, your radiation level will slowly decrease.  Taking Prussian blue tablets will help speed the process.  Get yourself a measuring device, as radiation is invisible and you may not know it that you are under its influence until it results in radiation sickness.",
+      "Though it is relatively rare, certain areas of the world may be contaminated with radiation.  It will gradually accumulate in your body, weakening you more and more.  While in radiation-free areas, your radiation level will slowly decrease; taking Prussian blue tablets will help speed this process.  Get yourself a measuring device, as radiation is invisible and you may not know that you are under its influence until it results in radiation sickness.",
```

Comment:
Medically speaking, it's not the radioactive material itself that causes radiation sickness and weakening, but the ionizing radiation the material emits. See the [Demon Core incident](https://en.wikipedia.org/wiki/Demon_core#Second_incident); the scientists were bathed in neutron radiation, but they didn't "accumulate" material in their bodies. Because the game says that simply _being present_ in irradiated areas will weaken you, this suggests that the radiation is gamma or neutron radiation; alpha or beta would require the player to ingest contaminated material.

Reply:
Prussian blue, however, works via helping remove radioactive material (definitely cesium and thallium, possible on a few others like rubidium). I am guessing the thinking is that radioactive areas may both irradiate you directly and have radioactive dust present. (Admittedly, breathing protection should really be blocking that part; I don't think it does, but I could be wrong.)

Reply:
Hm. Looking at the Wikipedia page for Prussian blue, it looks like this is a slight divergence from reality; Prussian blue can sequester heavy metals, but it doesn't seem like it's radioactivity-specific. Is this an acceptable break from reality?

Reply:
Strontium-90, which Prussian blue can also sequester according to some data, is both used in RTGs (which appear to be rather more common in the Cataclysm world) and a major concern in nuclear weapon fallout. Adding some other chelators to the game, and limiting Prussian blue to some fraction of radiation absorbed, may be a good idea in the long run. (I am assuming the various healing-related changes in the post-Cataclysm world are helping recover from radioactivity - it's instead promoting mutations!) I need to work on dialysis tubing - you would definitely want to replace it in a dialysis machine if you were using it to help extract chelated radioactive heavy metals! Chelation is hard on the kidneys...